live-build/misc/migration-scripts/dx_verify

```diff
@@ -141,12 +141,10 @@ function run_upgrade_verify() {
 		upgrade_verify_opts="-disableConsistentMdsZfsDataUtil"
 	local progress_low=$4
 	local progress_high=$5
-
-	[[ $(svcprop -p delphix/debug $MGMT_FMRI) == "true" ]] &&
-		delphix_debug="-Ddelphix.debug=true"
+	local debug_mode=$(svcprop -p delphix/debug $MGMT_FMRI)
```

Comment:
I suspect it's because the exit status of the command is overridden by the exit status of the creation the local variable. Compare
```
~$ f() { local foo=$(false) || echo blah; }; f
~$ f() { local foo; foo=$(false) || echo blah; }; f
blah
```